content/js/index.md

```diff
@@ -1,65 +1,142 @@
+The Skygear JS SDK is assumed to be run in ES6 runtime. If you wish to run it in
+an older runtime, you need to require the [polyfill plugin](https://babeljs.io/docs/usage/polyfill/)
+before importing the Skygear SDK.
 
-``` js
+``` javascript
 import 'babel-polyfill';
 import skygear from 'skygear';
 ```
 
-### Include in React Native projects
-You can also include Skygear JS SDK in your React Native project.
+## Include in your website
 
-``` js
-require('babel-polyfill');
-var React = require('react-native');
+Build the Skygear module first (make sure all the tests pass,
+if not please check your node version or report an issue):
 
-/* ... */
-
-var skygear = require('skygear');
+``` bash
+cd skygear
+npm test
+cd ..
 ```
 
-### Include in your website *(NOT YET IMPLEMENTED)*
-Place these lines before the body closing tag `</body>`. The SDK will be loaded from the skygear.io CDN.
+Then you can just include the javascript file `skygear/dist/bundle.js` to your
+html file like this:
 
 ``` html
-<script src="//cdn.skygear.io/polyfill.min.js"></script>
-<script src="//cdn.skygear.io/skygear.js"></script>
+<script src="/skygear/dist/bundle.js"></script>
 ```
 
+
+
+
+
+
+
+
+
+
+<br/><br/>
+<a name="intro-portal"></a>
+# Basics of Skygear JS SDK
+
+## Developer Portal
+
+To start using the Skygear JS SDK, you need to register your account and
+application at the Skygear [Developer Portal](https://portal-staging.skygear.io)
+website. After you registered, go to the **INFO** tab and copy your
+`Server EndPoint` and `API Key`, and right after where you include Skygear JS
+SDK configure the skygear container:
+
+``` javascript
+import skygear from 'skygear';
+// or in the browser with ECMAScript 5 just use window.skygear or skygear
+
+skygear.config({
+  'endPoint': 'https://<your-app-name>.staging.skygeario.com/', // trailing slash is required
+  'apiKey': '<your-api-key>'
+}).then((container) => {
+  console.log(container);
+}, (error) => {
+  console.error(error);
+});
+```
+
+## Container
+
+Container is the uppermost layer of skygear. In practice,
+`import skygear from 'skygear'` will give you a container instance at variable
+skygear. In most case you will only need one instance of container.
+
+## Database
+
+You will be provided with a private and a public database.
+
+- Everything in the private database is truly private, regardless of what access
+control entity you set to the record. In other words, each user has his own
+private database, and only himself have access to it.
+- Record saved at public database is by default public. Even without
+logging in, records in the public database can be queried (but not updated).
+To control the access, you may set different access control entity to the record.
+- The database objects can be accessed with `skygear.publicDB` and
+`skygear.privateDB`.
+
+
+
+
+
+
+
+
+
+
+<br/><br/>
```

Comment:
I put brbr before every title that is mentioned in the menu on the left to make the web page looks better and 10 lines to make the md file more readable. Maybe I shall remove all the brbr and the 10 lines?